teachertool/src/services/loggingService.ts

```diff
@@ -7,13 +7,12 @@ const formatName = (name: string) => {
     return name.toLowerCase().replace(/ /g, "_");
 }
 
-export const logError = (name: string, details: string) => {
-    pxt.tickEvent("teachertool.error", { name: formatName(name), message: details });
-    console.error(formatMessageForConsole(`${name}: ${details}`));
+export const logError = (name: string, details: string, props: pxt.Map<string | number> = {}) => {
```

Comment:
nit: rename `props` to `data` or something similar. When I think of props, I think of React props..